lib/util/pssh.js

```diff
@@ -109,4 +117,78 @@ shaka.util.Pssh = class {
       shaka.log.warning('Mismatch between box size and data size!');
     }
   }
+
+  /**
+   * Check if inidata contains Pssh
+   * @param {!Uint8Array} initData
+   * @param {!boolean=} extractMoov boolean to extract pssh from moov atom
+   * @returns {boolean}
```

Comment:
Oh hey, this should be `@return`, not `@returns`. Thanks, buildbot!

Reply:
Forgotten to run build after last commit. Did some fixes for lint errors and fixed a bug it caught